io.github.cataclysmbnteam.Cataclysm-BN.yml

```
@@ -0,0 +1,40 @@
+id: "io.github.cataclysmbnteam.Cataclysm-BN"
```

```suggestion
id: "org.cataclysmbn.CataclysmBN"
```

- we could use the shorter domain for simplicity, as the domain `cataclysmbn.org` is managed by https://github.com/cataclysmbnteam
- maybe no hyphen to keep up style with `org.cataclysmdda.CataclysmDDA`?

---

> * we could use the shorter domain for simplicity, as the domain `cataclysmbn.org` is managed by https://github.com/cataclysmbnteam

It says:
`There isn't a GitHub Pages site here.`

---

the site is accessible at https://docs.cataclysmbn.org subdomain.
@hfiguiere should we add `https://cataclysmbn.org/.well-known/org.flathub.VerifiedApps.txt` as well?

---

... somehow I never realized the docs page was just `docs.cataclysmbn.org`
So we do use it for hosting the docs at least

---

>     * maybe no hyphen to keep up style with `org.cataclysmdda.CataclysmDDA`?
the hyphen in the current one is due to the fact that it's there in the repo URL, but yes if we were to switch to using the cataclysmbn.org domain then the hyphen would indeed go away